client/store/planner.js

```diff
@@ -106,46 +105,42 @@ export const actions = {
     }
     storeToLocalStorage(state)
   },
-  async addCourse ({ commit, dispatch }, { payload, $api }) {
+  async addCourse({ commit, dispatch }, { payload, $api }) {
     await commit('addCourse', payload)
     await dispatch('saveSettings', { $api })
   },
-  async removeCourse ({ commit, dispatch }, { payload, $api }) {
+  async removeCourse({ commit, dispatch }, { payload, $api }) {
     await commit('removeCourseIndex', payload)
     await dispatch('saveSettings', { $api })
   },
-  async addPlan ({ commit, dispatch }, { payload, $api }) {
+  async addPlan({ commit, dispatch }, { payload, $api }) {
     await commit('addPlan', payload)
     await dispatch('saveSettings', { $api })
   },
-  async setTitleOf ({ commit, dispatch }, { payload, $api }) {
+  async setTitleOf({ commit, dispatch }, { payload, $api }) {
     await commit('setTitleOf', payload)
     await dispatch('saveSettings', { $api })
   },
-  async setCurPlan ({ commit, dispatch }, { payload, $api }) {
+  async setCurPlan({ commit, dispatch }, { payload, $api }) {
     await commit('setCurPlan', payload)
     await dispatch('saveSettings', { $api })
   },
-  async removePlan ({ commit, dispatch }, { payload, $api }) {
+  async removePlan({ commit, dispatch }, { payload, $api }) {
     await commit('removePlan', payload)
     await dispatch('saveSettings', { $api })
   },
-  async loadSettings (
-    {
-      commit,
-      rootState: {
-        auth: { loggedIn, user }
-      }
-    },
-    { $api }
-  ) {
-    const savedPlans = ls('plans')
-    const savedPlan = ls('plan')
+  async loadSettings({
+    commit,
+    rootState: { auth: { loggedIn, user } }
+  }, { $api }) {
+    const savedPlans = ls('plans'),
+    savedPlan = ls('plan')
     if (loggedIn) {
       const { plans, plan } = await $api.auth.getPlans()
-      await commit('setPlans', plans || savedPlans || DEFAULT_PLANS)
+      await commit('setPlans', plans || savedPlan || DEFAULT_PLANS)
       await commit('setCurPlan', plan || savedPlan || 0)
     } else {
+      const savedPlans = ls('plans')
```

Comment:
Aren't you just redeclaring the `savedPlans` again with what should be the same value? Why is this modified?

Reply:
you're right, but I don't remember doing this and I don't see the purpose of me doing it. I'll change it back

Reply:
I do not know why this is here either but without it, the download won't work. I get this error: 
![Screen Shot 2019-05-02 at 1 31 06 PM](https://user-images.githubusercontent.com/17994830/57105015-9cc1d900-6cde-11e9-93cd-ed7dd39a69e1.png)